app/controllers/v1/actions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class V1::ActionsController < V1::BaseController
+
+  def create
+    activity = Activity.find_by(template_id: params[:template_id])
+    action = Action.create(person: person_from_params, activity: activity)
+    if action.valid?
+      output = { success: true }
+    else
+      output = { error: 'could not create action' }
+    end
+    render json: output
+  end
+
+  private
+
+  def person_from_params
+    Person.find_by(email: params[:email]) || Person.find_by(phone: params[:phone])
```

Comment:
Oh, one thing that would be a nice to have later - allow creation of a person if none is found matching those inputs.